lib/amazonka/CHANGELOG.md

```diff
@@ -3,6 +3,33 @@
 ## [2.0.0](https://github.com/brendanhay/amazonka/tree/2.0.0)
 Released: **?**, Compare: [2.0.0-rc1](https://github.com/brendanhay/amazonka/compare/2.0.0-rc1...2.0.0)
 
+### Major Changes
+
+- The authentication code in `amazonka` got a full rewrite in PR [\#746](https://github.com/brendanhay/amazonka/pull/746).
+
+  - `newEnv` now takes a function of type `EnvNoAuth -> m Env`, which is to fetch credentials in an appropriate manner
+  - The `Credentials` type has been removed, you should instead use the following functions corresponding to the departed constructor. All are exported by `Amazonka.Auth`:
+
+    | Old Name          | New Name                     | Comment                                                                                                                                         |
+    |-------------------|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+    | `FromKeys`        | `fromKeys`                   | Explicit access key id/secret access key.                                                                                                       |
+    | `FromSession`     | `fromSession`                | As above, but with session token.                                                                                                               |
+    |                   | `fromTemporarySession`       | As above, but with token with expiry time. Likely not useful.                                                                                   |
+    |                   | `fromKeysEnv`                | Reads `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and optionally `AWS_SESSION_TOKEN`.                                                         |
+    | `FromEnv`         |                              |                                                                                                                                                 |
```

Comment:
Should these be in the same row?

Reply:
No, they're different. `FromEnv` took in the _names_ of the environment variables to use, whereas `fromKeysEnv`. I will push a clarification.